portfolio/src/main/webapp/script.js

```diff
@@ -56,10 +61,151 @@ function deleteComments(){
   generateComments(0);
 }
 
-function createMap() {
-  const map = new google.maps.Map(
+function dayMode() {
+  map = new google.maps.Map(
       document.getElementById('map'),
-      {center: {lat: 37.422, lng: -122.084}, zoom: 16});
+      {center: {lat: 37.7590, lng: -77.4800}, zoom: 7});
+
+    if(markersVisible){
+            markers();
+        }
+}
+
+function nightMode(){
+    // Styles a map in night mode.
+        map = new google.maps.Map(document.getElementById('map'), {
+          center: {lat: 37.7590, lng: -77.4800},
+          zoom: 7,
+          styles: [
+            {elementType: 'geometry', stylers: [{color: '#242f3e'}]},
+            {elementType: 'labels.text.stroke', stylers: [{color: '#242f3e'}]},
+            {elementType: 'labels.text.fill', stylers: [{color: '#746855'}]},
+            {
+              featureType: 'administrative.locality',
+              elementType: 'labels.text.fill',
+              stylers: [{color: '#d59563'}]
+            },
+            {
+              featureType: 'poi',
+              elementType: 'labels.text.fill',
+              stylers: [{color: '#d59563'}]
+            },
+            {
+              featureType: 'poi.park',
+              elementType: 'geometry',
+              stylers: [{color: '#263c3f'}]
+            },
```

Comment:
I would split these styles out into a new file that contains some relevant constants.  The details of this don't really matter, so it just clutters this code